routes/utils.js

```diff
@@ -430,3 +434,91 @@ function _freeId (objType, id) {
         }
     });
 }
+
+
+
+// generic function for send notif (and email)
+
+function get_mail_config () {
+    var MAIL_CONFIG = {};
+    // todo: more and more ugly init...
+    if (CONFIG[MAILER]) { MAIL_CONFIG = CONFIG[MAILER]; }
+    if (!MAIL_CONFIG.origin) {
+        logger.error('No email origin are configured !');
+    }
+    return MAIL_CONFIG;
+}
+
+async function gen_mail_opt (options, variables)
+{
+    var MAIL_CONFIG = get_mail_config();
+    // todo: check if each option exist and use default value
+    let name = options['name'];
+    let destinations = options['destinations'];
+    let subject = GENERAL_CONFIG.name + ' ' + options['subject'];
+
+    let message = 'Email message not found';
+
+    //find message
+    if (name && CONFIG.message[name]) {
+        message = CONFIG.message[name].join('\n');
+    } else {
+        logger.error('Email Message not found!');
```

Comment:
should we not return from fonction here?

Reply:
https://github.com/genouest/genouestaccountmanager/pull/166/commits/85ec1bdc1ec57c5022281b1b1304d9c89ebd0b98